### DISCUSSION TITLE
Puddletag review in PCLinuxOS Magazine

### DISCUSSION THREAD

**Discussion by sbstn87** [puddletag/puddletag#805] [General]
https://pclosmag.com/html/Issues/202306/page05.html

'If you are looking for an alternative to EasyTag that is Qt-based, Puddletag is an attractive option. Puddletag's distinctive spreadsheet-style GUI is a bit unfamiliar if you are accustomed to EasyTag's interface. However, it works well out of the box and if you only need to do the basics -- such as tagging and adding cover art to your music files -- you will soon feel at home with the program. If you are willing to delve deeper into its documentation, Puddletag's powerful bulk-editing capabilities become more apparent. I saw a comment from a Reddit user that Puddletag was able to easily handle 50,000 - 60,000 tracks in bulk tagging operations."

**Comment by sbstn87** [comment]
A Note on Puddletag's Flatpak

The most recent version of Puddletag, 2.2.0 (released in July 2022), is available on Flathub. Although I was able to successfully install the Flatpak on my PCLinuxOS system, I was disappointed that none of the online music databases/Tag Sources worked. When I tried to search, the app would throw error messages. I was unable to troubleshoot this glitch. Since metadata searching is an important feature to me, I uninstalled the Flatpak and went back to using ver. 1.2.0 from our official repository. (Another factor in my decision to delete the Flatpak was its huge size; after installation, it occupied nearly 5GiB of disk space).

**Comment by sandrotosi** [comment]
thanks for sharing! re flatpak, there's a reason we (the puddletag developers) do not recommend using it.

**Comment by sbstn87** [comment]
Can anything be done about the Flatpak version?

**Reply by sandrotosi** [reply]
we do not maintain the flatpak release, we provide no support for it, nor we recommend using it.
